src/uds/mod.rs

```diff
@@ -129,9 +111,9 @@ impl DiagProtocol<UDSErrorWrapper> for UDSProtocol {
         DiagPayload::new(UdsCommand::TesterPresent.into(), &[if response_required {0x00} else {0x80}])
     }
 
-    fn process_ecu_response(r: &[u8]) -> Result<Vec<u8>, (u8, UDSErrorWrapper)> {
+    fn process_ecu_response(r: &[u8]) -> Result<Vec<u8>, (u8, ByteWrapper<UdsError>)> {
```

Comment:
I would use `UdsErrorByte` everywhere instead of the `ByteWrapper` - as it is more explicit

Reply:
Fixed in ef6649084883d9f7a479f1665fb1879165daed29